java/viewshed-geoprocessing/src/main/res/layout/activity_main.xml

```diff
@@ -12,4 +12,30 @@
         android:layout_height="match_parent">
     </com.esri.arcgisruntime.mapping.view.MapView>
 
-</RelativeLayout>
+    <RelativeLayout
+        android:id="@+id/loadingView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/loading_view_bg"
+        android:clickable="true"
+        android:focusable="true"
```

Comment:
these may not be necessary

Reply:
@TADraeseke I think these are required to disable map interactions. When removed, the geoprocessing job fails when the map is tapped while the job is loading.